daemon/network.go

```diff
@@ -178,6 +178,10 @@ func (daemon *Daemon) SetupIngress(create clustertypes.NetworkCreateRequest, nod
 		if err := ep.Join(sb, nil); err != nil {
 			logrus.Errorf("Failed joining ingress sandbox to ingress endpoint: %v", err)
 		}
+
+		if err := sb.EnableService(); err != nil {
+			logrus.Errorf("Failed enabling service for ingress sandbox: %v", err)
```

Comment:
`WithError(err)` is not used here to be consistent with other logs in this file.